recipes/forestdb/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+class ForestDBConan(ConanFile):
+    name = "forestdb"
+    description = "ForestDB is a KeyValue store based on a Hierarchical B+-Tree."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ForestDB-KVStore/forestdb"
+    topics = ("forestdb", "kv store")
```

Comment:
```suggestion
    topics = ("kv-store", "wal", "mvcc")
```
Topics are used like keywords for searching. Avoid white space, otherwise, the search engine will understand it as different keywords.

Reply:
Whoops, yes. I copied this from a previous recipe I made. :+1: